firmware/const.h

```diff
@@ -2,28 +2,133 @@
 #define CONST_H
 
 #include <Arduino.h>
-
+//#define HOMEONSTART
+#define BAUDRATE 115200
+#define EASYCNCCONTROLLER
+#ifdef EASYCNCCONTROLLER
+#define EASYCNC_BUSY 4
+#define EASYCNC_IDLE 0
+#endif
+//#define DEBUG_MOVE
 //#define DEBUG_OUTPUT
+//#define DEBUG_EXECUTE
 //#define DEBUG_QUEUE
 //#define DEBUG_CURVE
 //#define DEBUG_SERIAL
 //#define DEBUG_POWER
 
-#define STEPS_PER_MM ( 1600l / 15 )
-#define UM_PER_STEP  ( 15000l / 1600l )
+// MOTHERBOARDS
+// 1 = RAMPS14
+// 2 = GRBL
+
+//#define CNCBOARD RAMPS14
+#define MOTHERBOARD 1
```

Comment:
#define a constant for RAMPS14 and GRBL so that we can use those instead of the magic numbers of "1" and "2".